Added information about environment variables used while running acceptance tests

kiranparajuli589 opened the pull request:
### Description
- update running acceptance tests description

### Related Issue
- fixes https://github.com/owncloud/core/issues/38918

mmattel commented:
I have fixed some minor text things (camel case, ect) and squashed the commits.

Please tell to which branches this needs backporting or does it stay in master only.

kiranparajuli589 commented:
> I have fixed some minor text things (camel case, ect) and squashed the commits.
> 
> Please tell to which branches this needs backporting or does it stay in master only.

I've rebased the branch with master again. I think it should also be backported to the `10.8` branches including the current master

phil-davis commented on modules/developer_manual/pages/testing/acceptance-tests.adoc:
```diff
@@ -269,6 +286,47 @@ When the acceptance tests are run, the user names and attributes will be replace
 describing the outcome of your testing.
 
+=== Running Acceptance Tests Using Part System
+The part system allows us to divide the test run without knowing how many or which suites are available. Filter tags can also be used to run targeted parts after divisions.
```
```suggestion
The part system allows us to divide the test run without knowing how many or which suites are available. Filter tags can also be used to run targeted scenarios after divisions. Multiple test suites are grouped together in each part. For example, if there are 26 test suites to be grouped into 10 parts then 2 or 3 test suites will be run in each part. This functionality is most useful in CI. The CI can split the test suites into as many pipelines as is appropriate without needing to know the actual names of the test suites.
```

phil-davis commented on modules/developer_manual/pages/testing/acceptance-tests.adoc:
```diff
@@ -269,6 +286,47 @@ When the acceptance tests are run, the user names and attributes will be replace
+|The part-number to test
+|===
+
+Execute the tests by setting the above environments and running the usual make command.
```
```suggestion
Execute the tests by setting the above environment variables and running the usual make command.
```

phil-davis commented on modules/developer_manual/pages/testing/acceptance-tests.adoc:
```diff
@@ -269,6 +286,47 @@ When the acceptance tests are run, the user names and attributes will be replace
+
+**2. part argument**
+
+If `run.sh` is used directly to run acceptance tests, part system can be achieved using `--part` flag. The script below divides the test suite into five parts and runs just the first one.
```
```suggestion
If `run.sh` is used directly to run acceptance tests, the part system can be achieved using the `--part` flag. The script below divides the test suite into five parts and runs just the first one.
```

phil-davis commented:
Backport 10.8 #3836
